styles.css

```diff
@@ -44,13 +50,71 @@ header nav ul {
 }
 header nav ul li a {
   text-decoration: none;
-  color: #fff;
+  color: #fff;  
 }
-body #post-section {
-position: absolute;
+header nav ul li a:hover {
+	text-shadow: 2px 2px #E427DD;
+	}
+#barradepost {
```

Comment:
please lets follow a pattern. every variable, id, class name... anything should be in English